docs/source/api_reference.rst

```diff
@@ -133,6 +133,12 @@ Pipeline Base Classes
     TimeSeriesBinaryClassificationPipeline
     TimeSeriesMulticlassClassificationPipeline
     TimeSeriesRegressionPipeline
+    GeneratedPipelineBinary
```

Comment:
Curious, do you think it make sense to include these in our API reference? I ask because I'm not really sure they provide too much value on their own lol. Are these classes basically just implementation detail for enabling pickling?

Reply:
Specifically, I ask this because if a user were to interface with `GeneratedPipelineBinary` as they get different pipelines, they'd notice that all of its attributes are changing. That's a bit odd. I'd argue we'd even want to make this private 🤷

Reply:
Yeah, it's probably not necessary. I added it just in case, but I can remove it!